Analysis/include/EventInfo.h

```diff
@@ -139,6 +139,15 @@ class EventInfoBase {
         return selected_pair;
     }
 
+    std::set<size_t> GetSelectedBjetIndices()
```

Comment:
Better return std::array<size_t, 2> here. Also add const modifier to the method.